GridMvc/Html/IGridHtmlOptions.cs

```diff
@@ -104,6 +104,12 @@ public interface IGridHtmlOptions<T> : IHtmlString
         /// </summary>
         IGridHtmlOptions<T> WithGridItemsCount(string formatString = null);
 
+        /// <summary>
+        ///     Disable default paging
```

Comment:
This should be described a bit more in detail.
What does disabling default paging mean, what are the behavior changes/consequences?

It is not obvious from this description, that the paging navigation would be displayed nonetheless, but all items of the `IQueryable` will be displayed without restricting them to the current page.